ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,60 @@
-﻿namespace ConsoleApp1
+﻿
+namespace ConsoleApp1
 {
 	internal class Program
 	{
 		static void Main(string[] args)
 		{
+            byte b1 = 0;
+            short s1 = 0;
+            int i1 = 0;
+            long l1 = 0;
+            bool b2 = false;
+            char c1 = 'a';
+            float f1 = 0.0f;
+            decimal d1 = 0.0m;
+            string s2 = String.Empty;
```

Comment:
fix formatting

Reply:
Please specify which formatting you are talking about